stoplight.js

```diff
@@ -0,0 +1,13 @@
+alert ("working")
+
+$(document).ready(function () {
+    ('red').click(function(e) {
```

Comment:
None of these selectors will work if you don't use `$` before them.

Should be `$("red").click( ... )`